tests/root_chain/contracts/root_chain/test_root_chain.py

```diff
@@ -95,41 +110,57 @@ def test_start_exit(t, root_chain, assert_tx_failed):
     priority3 = child2_blknum * 1000000000 + 10000 * 0 + 0
     sigs = tx3.sig1 + tx3.sig2 + confirmSig1 + confirmSig2
     # Double input exit
-    exitId3 = child2_blknum * 1000000000 + 10000 * 0 + 0
-    root_chain.startExit(exitId3, tx_bytes3, proof, sigs, sender=key)
-    assert root_chain.getExit(priority3) == ['0x' + owner.hex(), 100, exitId3]
+    utxoPos3 = child2_blknum * 1000000000 + 10000 * 0 + 0
+    root_chain.startExit(utxoPos3, tx_bytes3, proof, sigs, sender=key)
+    assert root_chain.getExit(priority3) == ['0x' + owner.hex(), 100]
 
 
-def test_challenge_exit(t, u, root_chain):
+def test_challenge_exit(t, u, root_chain, assert_tx_failed):
```

Comment:
This should definitely be broken down into some components but maybe that's for another PR.